UdpHosts/GameServer/Entities/Character/MovementStateContainer.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameServer.Entities.Character
+{
+    /// <summary>
+    /// Allows eased access to the several flags
+    /// </summary>
+    internal class MovementStateContainer
```

Comment:
Is the "Container" suffix coming from Firefall naming?  
To me it's a bit confusing, as it's not really a container in the sense of a collection, but maybe that's just my C++ bias 😄

Reply:
The `Container` suffix was chosen as a generic suffix without any implicit semantics in mind.
In essence, the class' purpose is to contain flags and accessors, so `Container` seemed like the correct choice. 
If there's a more fitting term available, we should definitely switch to it, but `Collection` is definitely out of question imo, haha 😁